classes/packagefile.php

```diff
@@ -158,9 +158,15 @@ public static function download_wrapper($scormremote, $filename, $clientid) {
                 $resource->attributes()->href // The original file path.
             );
 
+            $params = [
+                'loglevel' => get_config('mod_scormremote', 'debugloglevel')
+            ];
+
+            $query = http_build_query($params);
+
             $templatedata = [
                 'datasource'       => $datasource,
-                'jssource'         => $CFG->wwwroot . '/mod/scormremote/amd/src/layer2.js',
+                'jssource'         => $CFG->wwwroot . '/mod/scormremote/amd/src/layer2.js?' . $query,
```

Comment:
what happens to this param? I can't see any code linked to the actual logging inside layer2.js or layer3.js?

layer2.js has a debug which is boolean, but layer3 has the levels and I assume both need to be touched in some way